Buildings/Fluid/Dehumidifiers/Desiccant/Data/Default.mo

```diff
@@ -0,0 +1,51 @@
+within Buildings.Fluid.Dehumidifiers.Desiccant.Data;
+record Default =
```

Comment:
you may rename it as `EnergyPlus`. See `Buildings.Fluid.HeatPumps.Data.EquationFitReversible`

Reply:
Renamed the model as suggested.